.github/workflows/format-lint-test.yaml

```diff
@@ -0,0 +1,31 @@
+name: Format, Lint, Test
+
+on:
+  push:
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js
+      uses: actions/setup-node@v3
```

Comment:
There is a `actions/setup-node@v4` as well should we use the latest version or does it not matter at all?

Reply:
If official Github suggests this then it's not really necessary to change that I guess.